ui/app/templates/vault/cluster/recovery/snapshots.hbs

```diff
@@ -1,6 +0,0 @@
-{{!
-  Copyright (c) HashiCorp, Inc.
-  SPDX-License-Identifier: BUSL-1.1
-}}
-
-{{outlet}}
```

Comment:
Same comment here as above, this is useful if there's something in common all of these routes share, but I don't think there is?